src/Passes/README.md

```diff
@@ -0,0 +1,322 @@
+# Q# Passes for LLVM
+
+This library defines LLVM passes used for analysing, optimising and transforming the IR. The Q# pass library is a dynamic library that can be compiled and ran separately from the
+rest of the project code. While it is not clear whether this possible at the moment, we hope that it will be possible to write passes that enforce the QIR standard.
+
```

Comment:
Consider making the fragment "LLVM passes" a web link to something like [this](https://llvm.org/docs/WritingAnLLVMPass.html#introduction-what-is-a-pass). Such that the reader immediately understands that the "LLVM passes" is a special term with a strict meaning (and gets the right intro/tutorials before diving into the topic).

Reply:
Done!

Reply:
FYI: Doing that _for the first-most occurrence only_ of the "LLVM passes", was sufficient. ;-)